libs/core/src/components/pds-chip/pds-chip.scss

```diff
@@ -2,22 +2,22 @@
   --box-shadow-focus: 0 0 0 1px #ffffff, 0 0 0 3px var(--pine-color-purple-300);
 
   --color-background-accent-dot: var(--pine-color-purple-600);
-  --color-background-accent-hover: var(--pine-color-purple-200);
+  --color-background-accent-hover: var(--pine-color-purple-300);
```

Comment:
we should be using `--pine-chip-color-*-hover`. This will need to be added to `libs/core/src/global/styles/tokens/components/chip.json`